cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -48,7 +48,7 @@ var (
 	argEtcdMutationTimeout = flag.Duration("etcd_mutation_timeout", 10*time.Second, "crash after retrying etcd mutation for a specified duration")
 	argEtcdServer          = flag.String("etcd-server", "http://127.0.0.1:4001", "URL to etcd server")
 	argKubecfgFile         = flag.String("kubecfg_file", "", "Location of kubecfg file for access to kubernetes service")
-	argKubeMasterUrl       = flag.String("kube_master_url", "https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}", "Url to reach kubernetes master. Env variables in this flag will be expanded.")
+	argKubeMasterUrl       = flag.String("kube_master_url", "", "Url to reach kubernetes master. Env variables in this flag will be expanded.")
```

Comment:
Why change this flag? If someone wants to use RO service, let them manually override this flag.